modules/applications/pages/ui-builder/common-apis-to-use.adoc

```diff
@@ -0,0 +1,101 @@
+= Common APIs to use
+:page-aliases: applications:common-apis-to-use.adoc
+:description: Explore the most commonly used APIs and JS objects for building an app like HR Management in UIB.
+
+{description}
+
+Bonita APIs combined with JS objects allow for building all kind of applications, such as: account management dashboard, custom CRM, HR Management tool, invoice processing dashboard, and many more.
+
+
+image:ui-builder/guides/hr-mgmt-app.png[hr-mgmt-app]
```

Comment:
Since we're removing references to the HR management app, should we keep this picture? I don't have a strong opinion on this